### PR TITLE
Shindo to Minitest: network

### PR DESCRIPTION
**Pull request by gildub**
Migrates network requests and models
    
One test randomly failing has been marked as skipped using Minitest::Test::UNIT_TESTS_CLEAN:
- test/requests/network/port_tests.rb
    
Note: This is not using subfolder 'openstack' anymore

### CODE REVIEW

File: fog-openstack.gemspec [hunk 1]
```diff
@@ -33,4 +33,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest'
   spec.add_development_dependency 'vcr'
   spec.add_development_dependency 'webmock',  '~> 1.24.6'
+  spec.add_development_dependency 'pry-byebug'
```
Comment (petrblaho) [inline fog-openstack.gemspec]: Is this needed?

Comment (gildub) [inline fog-openstack.gemspec]: Removed it! Thanks

File: test/models/network/floating_ip_tests.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      @instance.port_id = 'p0000000-0000-0000-0000-000000000000'
+      @instance.update.port_id.must_equal "p0000000-0000-0000-0000-000000000000"
+    end
+
+    it "#associate" do
+      port_id = 'p0000000-0000-0000-0000-000000000000'
+      fixed_ip_address = '8.8.8.8'
+      @instance.associate(port_id, fixed_ip_address)
+      @instance.port_id.must_equal port_id
+      @instance.fixed_ip_address.must_equal fixed_ip_address
```
Comment (petrblaho) [inline test/models/network/floating_ip_tests.rb]: I would like to see this split so there is only one assert per `it` block.

File: test/models/network/floating_ip_tests.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    end
+
+    it "#disassociate" do
+      fixed_ip_address = '8.8.8.8'
+      @instance.disassociate(fixed_ip_address)
+      @instance.port_id.must_equal nil
+      @instance.fixed_ip_address.must_equal nil
```
Comment (petrblaho) [inline test/models/network/floating_ip_tests.rb]: DITTO

**Comment by gildub** [comment]
@petrblaho, per your latest comment I have created a test block for each assertion. Thanks

**Pull request merged**